WahooFitness/Parse-SDK-iOS-OSX
# Parse upstream update

Merge upstream changes from parse-community/Parse-SDK-iOS-OSX master branch. This is needed to compile successfully on Xcode 12.

To test this:

On the legacy SUF app project:
1. Using Xcode 12 checkout the master branch and try to build. It should fail
2. Checkout the branch named upgrade-to-xcode12 and try to build. It should succeed
3. Verify you can log in and play workouts
